doc/source/train/user-guides/results.rst

```diff
@@ -0,0 +1,91 @@
+Inspecting Training Results
+===========================
+
+The return value of your :meth:`Trainer.fit() <ray.train.trainer.BaseTrainer.fit>`
+call is a :class:`~ray.air.result.Result` object.
+
+The :class:`~ray.air.result.Result` object contains, among others:
+
+- The last reported metrics (e.g. the loss)
+- The last reported checkpoint (to load the model)
+- Error messages, if any errors occurred
+
+Last reported metrics
+---------------------
+
+Use :py:attr:`Result.metrics <ray.air.result.Result.metrics>` to retrieve the
```

Comment:
These don't seem to be rendering properly.

Reply:
Yeah I've tried a few things and couldn't find the right keyword so far

Reply:
Found it - for the attributes `ray.air.Result.metrics` etc has to be used (not `result.Result`...)